rr/pc_chair_guide.md

```diff
@@ -0,0 +1,58 @@
+# So You've Become The RR Track Chair ...
+
+Things run a bit differently in Registered Report tracks that want to provide EMSE Journal in principle acceptances (IPA). In a typical track, with the guidance of the GC and technical PC chairs, you and co-chairs have a lot of autonomy on how to run things. 
+
+In RR tracks, while the guidance of the other organizers is important, we add this second phase with the EMSE Journal. This means the RR chairs have to 
+1. manage the conference submission and review process, and any conference activities (camera-ready deadlines, program scheduling)
+2. track the ongoing and in progress studies
+3. act as point of contact for authors, and explain to authors how to submit the Phase 2 report/paper and
+4. assign reviewers and manage the journal reviews and decision (via Springer's Editorial Manager)
+
+## Background on RR
+You can read about motivation and process [in a paper we wrote](https://arxiv.org/abs/2302.03649v1). 
+
+## Journal Policies and Process
+EMSE Journal has [official policies](https://github.com/emsejournal/openscience/blob/master/rr/rr_policies.md) on RR. 
+
+Please contact Neil and Teresa (the journal RR editors) before making any major changes or policy decisions on the RR process. 
+
+## Balancing constraints: 
+- EMSE and conference quality should be paramount. We don't want to accept, even provisionally, papers that don't have the ability to make an excellent contribution to the journal and conference.
+- Author benefits: the RR track offers two main benefits. One is the immediate feedback on the research protocol. This is independent of the review process outcomes, and so *even rejected papers get this benefit*. Don't discount this! 
+- The second is the "in principle acceptance" to EMSE. We read this as equivalent to an expectation of **only minor revisions** at the second phase (and possibly "minor editorial"). IPA should NOT be granted to papers that will likely require major revisions. This undermines the benefit of RR. 
```

Comment:
Neil, I'm not sure what you mean with "minor editorial" here!?

Reply:
I thought there was "minor" and then "editorial" (spelling etc.) but maybe I have my journals confused :)

Reply:
We do have ~"Accepted with minor edits" but I don't think we need to mention that here.

Reply:
fixed